gauge/table.go

```diff
@@ -107,6 +107,17 @@ func (table *Table) AddHeaders(columnNames []string) {
 	}
 }
 
+func (table *Table) AddHeaders1(columnNames []string) {
```

Comment:
Can try reusing the function [AddHeaders](https://github.com/getgauge/gauge/blob/377ebe0806ff0f6eb7830023f749d97e14bdce03/gauge/table.go#L99) itself here?